### PR TITLE
Improve UX of blocks with animations applied and playing

### PR DESCRIPTION
**Pull request by jrtashjian**
### Description
Fixes the new animations UX by hiding all block toolbars while animations play and only applies the animation classes to the `wrapperProps` when the animation attribute has changed.

### Screenshots
Video: https://drive.google.com/file/d/18wqIDVhv9LmqpA4bX6e9B5pjpSGErGCE/view?usp=sharing

Screen Shot 2020-10-07 at 6 09 54 PM
Screen Shot 2020-10-07 at 6 10 07 PM
Screen Shot 2020-10-07 at 6 10 23 PM

### Types of changes
Bug fix

### How has this been tested?
Manually

### Checklist:
- [x] My code is tested
- [x] My code has proper inline documentation 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/extensions/animation/controls.js [hunk 1]
```diff
@@ -24,15 +22,17 @@ import {
...
 import { withSelect } from '@wordpress/data';
 
 function PreviewAnimationPopover( { hoveredAnimation, selected } ) {
-	if ( ! hoveredAnimation ) return null;
+	if ( ! hoveredAnimation ) {
+		return null;
+	}
 
-	const block = _.cloneDeep(selected);
+	const block = _.cloneDeep( selected );
```
Comment (jasonlemay) [inline src/extensions/animation/controls.js:35]: Are you using a linter for that? I'm sorry for all of the lint fixes 😓

**Comment by jasonlemay** [comment]
@jrtashjian Clever! That's awesome 👍

**Review by jasonlemay — APPROVED** [review]

**Review by AnthonyLedesma — APPROVED** [review]
This works perfectly! Great job @jrtashjian.

File: src/extensions/animation/controls.js (hunk 1 above)
Comment (jrtashjian) [reply src/extensions/animation/controls.js:35]: Yeah, it's all automated in my IDE. I just run the formatter and commit the changes :)

**Pull request merged**